core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -875,6 +875,10 @@ static function (array $allowedProperties): array {
                         )
                     ->end()
                 ->end()
+                ->integerNode('max_header_size')
+                    ->info('Do not increase this value beyond the allowed response headers size of your web server, as this will result in a 500 server error.')
```

Comment:
```suggestion
                    ->info('Do not increase this value beyond the allowed response header size of your web server, as this will result in a 500 server error.')
```